Content.Client/GameObjects/Components/Cargo/CargoConsoleBoundUserInterface.cs

```diff
@@ -137,7 +138,7 @@ private void AddOrder()
                 _orderMenu?.Requester.Text ?? "",
                 _orderMenu?.Reason.Text ?? "",
                 _product?.ID ?? "",
-                _orderMenu?.Amount.Value ?? 0));
+                Math.Min((_orderMenu?.Amount.Value ?? 0), ShuttleCapacity.MaxCapacity)));
```

Comment:
clientside validation is basically useless, please do it in server. but also keep it in here ofc

Reply:
Yup, it is done on the server as well

Reply:
Should probably make AddOrder return a bool instead and not send an order if the amount is over the amount. The bool would be so that it doesn't close the UI if it fails.